### PR TITLE
Hack duration.

### PR DESCRIPTION
**Pull request by vianneyPL**

### CODE REVIEW

File: src/datasource.js [hunk 1]
```diff
@@ -107,11 +109,19 @@ export default class Datasource {
     }
   }
 
+  transformDate = value => {
+    let d = Date.parse(value)
+    // handle timestamp as duration
+    if (d < this.maxDurationYear) {
+      return this.maxDurationYear - d
```
Comment (mkurdej) [inline src/datasource.js:116]: That's incorrect, it should be `return d - this.epochYear` (where epoch is 1970-01-01).

Comment (mkurdej) [reply src/datasource.js:116]: And probably you should verify that `d` is not before epoch.

**Comment by mkurdej** [comment]
Oh, and you can add a comment about your hack (somewhere near the code that checks `maxDurationYear`).

File: src/datasource.js (hunk 1 above)
Comment (vianneyPL) [reply src/datasource.js]: fixed.

**Pull request closed without merging**